boards/jiminy-mega256rfr2/include/board.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2016 RWTH Aachen, Josua Arndt
+ * Copyright (C) 2014 Freie Universität Berlin, Hinnerk van Bruinehsen
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_jiminy-mega256rfr2 Jiminy- Mega256rfr2
+ * @ingroup     boards
+ * @brief       Board specific files for the Jiminy Mega 256rfr2 board.
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the Jiminy Mega 256rfr2 board.
+ *
+ * @author      Hinnerk van Bruinehsen <h.v.bruinehsen@fu-berlin.de>
+ * @author      Josua Arndt <jarndt@ias.rwth-aachen.de>
+ * @author      Steffen Robertz <steffen.robertz@rwth-aachen.de>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*Fastest Baudrate possible for board*/
+#define UART_STDIO_BAUDRATE (250000U)
+
+/**
+ * @name   LED pin definitions and handlers
+ * @{
+ */
+#define LED_PORT            PORTB
+#define LED_PORT_DDR        DDRB
+
+#define LED_BLUE_PIN        GPIO_PIN(1, 5)
+#define LED_GREEN_PIN       GPIO_PIN(1, 6)
+#define LED_RED_PIN         GPIO_PIN(1, 7)
+
+#define BLUE            (1 << DDB5)
+#define GREEN           (1 << DDB6)
+#define RED             (1 << DDB7)
+/** @} */
+
+/**
+ * @name Define the interface to the AT86RF231 radio
+ *
+ * {spi bus, spi speed, cs pin, int pin, reset pin, sleep pin}
+ * @{
+ */
+#define AT86RF2XX_PARAMS_BOARD      {}
```

Comment:
So the radio is not configured ? Does it work ? If not, I guess you can remove the dependency and this config.

Reply:
at86rf2xx has default values. Transciever is SOC no SPI. Made an empty config to get rid of the default values. I will add a better fitting comment.

https://github.com/RIOT-OS/RIOT/blob/f3b277dcc4a65fdb56a53380b14a589c1786b603/drivers/at86rf2xx/include/at86rf2xx_params.h#L63

Reply:
We decided to remove this part until the PR of the transceiver.